src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -7,6 +7,39 @@ const IncludeGrid = styled(GridItemConstrainedMedium)`
   display: grid;
 `;
 
+const PocketTapArea = styled.div`
+  &&& {
```

Comment:
Looked this up here, is there a reason you need override styles: https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity ?

I wouldn't expect us to need to do this as I'd expect styles to be encapsulated as we are using CSS in JS.

Reply:
I used this to override the `i-amphtml-element>[overflow]` attribute selector which Amp adds to the amp-iframe. The attribute selector in the amp-iframe seems to be injected after the styled-component, so overrides the style, unless the ampersands are used to make the PocketTapArea styles more specific.

Reply:
@simonsinclair has showed me a cleaner approach 👍